### PR TITLE
N14: Add audir 0.1.0 section

### PR DESCRIPTION
**Pull request by msiglreith**
#278

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [audir]
+
+`audir` is a low level audio library supporting Windows (WASAPI), Linux (Pulse) and Android (OpenSLES & AAudio).
+
+It aims at provide a minimal and mostly unsafe but feature-rich API on top of common audio backends with focus on gaming applications.
+The initial release version `0.1.0`provides basic recording and playback support for all available backends, including a small music player example!
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:74]:
```suggestion
The initial release version `0.1.0` provides basic recording and playback support for all available backends, including a small music player example!
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks fine, just needs linter errors fixing (due to lines being more than 80 characters long).

**Comment by msiglreith** [comment]
Thanks for reviewing!

**Review by 17cupsofcoffee — APPROVED** [review]
Perfect, thanks for amending!

**Review by ozkriff — APPROVED** [review]
:rocket:

**Comment by ozkriff** [comment]
Awesome, merged!

**Pull request merged**